components/RecentEpisodes.vue

```diff
@@ -19,55 +21,82 @@ onMounted(async () => {
 </script>
  
 <template>
-  <div v-if="dataLoaded">
-    <div class="latest-episode grid justify-content-center">
+  <div>
+    <div v-if="dataLoaded" class="latest-episode grid justify-content-center">
       <div class="col-12 xl:col-8 p-0">
-        <img class="latest-episode-image" :src="episodes[0].attributes['image-main'].url" :alt="episodes[0].attributes['image-main']['alt-text']" />
+        <!-- <nuxt-link :to="`/episodes/${episodes[0].attributes.slug}`">
+          <img class="latest-episode-image" :src="episodes[0].attributes['image-main'].url" :alt="episodes[0].attributes['image-main']['alt-text']" />
+        </nuxt-link>-->
+        <v-image-with-caption
+          :image="episodes[0].attributes['image-main'].url"
+          :imageUrl="`/episodes/${episodes[0].attributes.slug}`"
+          :alt="episodes[0].attributes['image-main']['alt-text']"
+          :max-width="episodes[0].attributes['image-main'].w"
+          :max-height="episodes[0].attributes['image-main'].h"
+          :ratio="[4, 3]"
+        />
       </div>
       <div class="latest-episode-content col-12 xl:col-4 p-8">
-        <p class="uppercase mb-2">Latest Episode</p>
-        <h2 class="mb-4"><nuxt-link :to="episodes[0].attributes.url" v-html="episodes[0].attributes.title" class="latest-episode-title" /></h2>
-        <p v-html="episodes[0].attributes.tease" class="mb-5" />
-        <p class="latest-episode-podcasts"><apple-podcasts /> Apple Podcasts</p>
+        <p class="latest-episode-header mb-2">Latest Episode</p>
+        <h2 class="mb-4">
+          <nuxt-link
+            :to="`/episodes/${episodes[0].attributes.slug}`"
+            v-html="episodes[0].attributes.title"
+            class="latest-episode-title"
+          />
+        </h2>
+        <p v-html="episodes[0].attributes.tease" class="latest-episode-tease mb-5" />
+        <p class="latest-episode-podcasts">
+          <apple-podcasts />Apple Podcasts
+        </p>
       </div>
     </div>
-    <div class="recent-episodes">
+    <latest-episode-skeleton v-else />
+    <div class="recent-episodes container">
       <div class="flex justify-content-between">
-        <h2 class="mb-4">Recent Episodes</h2>
+        <h3 class="mb-4">Recent Episodes</h3>
         <nuxt-link class="all-episodes" to="/episodes">All Episodes</nuxt-link>
```

Comment:
these buttons are base Button components. You can use the VFlexibleLink comp with a button like this... with the raw prop, which resolves all the hover garbage and just let's the button be a button but with nuxt-link routing

<v-flexible-link raw to="episodes">
    <Button class="p-button-rounded" label="Recent Episodes" />
</v-flexible-link>

you may need to add "p-button-sm" to the Button to make it a smaller button (less padding)

and you can then remove the extra styling for this particular button

Reply:
I made this change but the button isn't styled at all - not sure if you are adding it with your PR but I can go back and fix it if not